compiler/runtime/runtime32_specific.c

```diff
@@ -2,13 +2,14 @@
 #include <stdbool.h>
 #include "xoroshiro128plus.h"
 
-void __cxa_throw(void*, void*, void*);
+void __cxa_throw(void*, uint32_t, uint32_t);
```

Comment:
Is there a spec or some docs for what this does with llvm targeting wasm? I wonder since this doesn't seem to correspond to the usual C++ ABI:
```
    void __cxa_throw (void *thrown_exception, std::type_info *tinfo, void (*dest) (void *) );
```
>The arguments are:
>- The address of the thrown exception object (which points to the throw value, after the header, as specified above).
>- A std::type_info pointer, giving the static type of the throw argument as a std::type_info pointer, used for matching potential catch sites to the thrown exception.
>- A destructor pointer to be used eventually to destroy the object.